util/config/config.go

```diff
@@ -431,3 +436,33 @@ func (config *Configuration) CheckPorts(myIP string) (bool, error) {
 	}
 	return true, nil
 }
+
+func OpenConfigFile() ([]byte, error) {
+	var file []byte
+
+	configFile := ConfigFile
```

Comment:
This get config file name part duplicates in OpenConfigFile() and WriteConfigFile(), we can write a common function for this part, such that later when we need to change it, we only need to change it once